cypress/e2e/pages/vaults/index.spec.ts

```diff
@@ -97,18 +93,22 @@ context("/vaults on mobile", () => {
     });
 
     it("should have vault mobile cards", () => {
-      cy.findAllByTestId("VaultMobileCard").within(() => {
-        cy.findByTestId("VaultMobileCard.VaultStatus").should("be.visible");
-        cy.findByTestId("VaultMobileCard.VaultID").should("be.visible");
-        cy.findByTestId("VaultMobileCard.Toggle").should("be.visible");
-        cy.findByTestId("VaultMobileCard.Toggle").click();
-        cy.findByTestId("VaultMobileCard.Loans").should("be.visible");
-        cy.findByTestId("VaultMobileCard.LoansValue").should("be.visible");
-        cy.findByTestId("VaultMobileCard.Collateral").should("be.visible");
-        cy.findByTestId("VaultMobileCard.CollateralValue").should("be.visible");
-        cy.findByTestId("VaultMobileCard.MinCollateralizationRatio").should(
-          "be.visible"
-        );
+      cy.findAllByTestId("VaultMobileCard").each(($el) => {
+        cy.wrap($el).within(() => {
+          cy.findByTestId("VaultMobileCard.VaultStatus").should("be.visible");
+          cy.findByTestId("VaultMobileCard.VaultID").should("be.visible");
+          cy.findByTestId("VaultMobileCard.Toggle").should("be.visible");
+          cy.findByTestId("VaultMobileCard.Toggle").click();
+          cy.findByTestId("VaultMobileCard.Loans").should("be.visible");
+          cy.findByTestId("VaultMobileCard.LoansValue").should("be.visible");
+          cy.findByTestId("VaultMobileCard.Collateral").should("be.visible");
+          cy.findByTestId("VaultMobileCard.CollateralValue").should(
+            "be.visible"
```

Comment:
Can we make this into single line like this cy.findByTestId("CollateralCard.AssetIcon").should("be.visible"); to maintain its readability?